modules/land.js

```diff
@@ -8,11 +9,65 @@ function Land()
 }
 
 Land.prototype = {
-  draw: function(ctx)
+  draw: function()
```

Comment:
i think 
land.update: function(){
this.removeDeadOrganisms()
this.draw()
}

land.draw(){
this.ctx blah
this.drawOrganisms()
this.drawFoods()
}